pkg/entrypoint/entrypointer.go

```diff
@@ -91,6 +91,14 @@ func (e Entrypointer) Go() error {
 			// An error happened while waiting, so we bail
 			// *but* we write postfile to make next steps bail too.
 			e.WritePostFile(e.PostFile, err)
+			output = append(output, v1alpha1.PipelineResourceResult{
```

Comment:
how about just doing this once, when `output` is initialized?

```go
// line 88
output := []v1alpha1.PipelineResourceResult{
  {
    Key: "StartedAt",
    Value: time.Now().Format(time.RFC3339),
  }
}
```

Could also remove lines 109-112 then as well.

Reply:
I think we can not initialize it at one place, the `time.Now()` should after `wait` "wait files",
and the `return`s are in different code block, so the `time.Now()` is different.